workflows/mag.nf

```diff
@@ -561,7 +561,7 @@ workflow MAG {
         * DAS Tool: binning refinement
         */
 
-        if ( params.refine_bins_dastool && !params.skip_metabat2 && !params.skip_maxbin2 ) {
+        if ( params.refine_bins_dastool && !params.skip_metabat2 && !params.skip_maxbin2 && !params.skip_concoct ) {
```

Comment:
Do I understand this line correct that refinement with dastool will only be used when all three binners were not skipped? While I agree that it is a good idea to use all 3 binners for refinement, shouldnt it be possible to also use refinement with only 2 binners?

Reply:
Done